src/plugin_discovery/plugin_spec/plugin_spec.js

```diff
@@ -48,6 +48,12 @@ export class PluginSpec {
    * directory for this plugin. The final directory must have the name "public",
    * though it can be located somewhere besides the root of the plugin. Set
    * this to false to disable exposure of a public directory
+   * @param {String|False} [opts.styleSheet] path to the stylesheet within
```

Comment:
Thoughts on the naming of these two options? Would `styleSheetPath` and `scssPath` be better?